pkg/apis/ceph.rook.io/v1beta1/version.go

```diff
@@ -0,0 +1,38 @@
+/*
+Copyright 2018 The Rook Authors. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package v1beta1
+
+const (
+	Luminous             = "luminous"
```

Comment:
Do you think we are likely to need more specific information about the version that the major codename?

Reply:
To start with we don't, but I'm anticipating we will have things come up that will be difficult to manage without detecting the minor release version. For example, we need to ensure we have a supported version of ceph-volume, which isn't found until a recent nautilus build and will be backported soon to luminous and mimic. I might need to do minor version checking for that.

Reply:
It seems like that can probably be done in the future. There isn't so much code related to this that it can't be reworked when that needs to happen.